pages/views-11ty/index.html

```diff
@@ -0,0 +1,193 @@
+            </li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex">Array.prototype.findIndex</a>
+            </li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes">Array.prototype.include</a>
+            </li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is">Object.is</a>
+            </li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise">ES6
+              Promises</a></li>
+          </ul>
+        </div>
+        <div class="col">
+          Web API
+          <ul>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent">Custom Events</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/Node/isConnected">Node.isConnected</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/Element/matches">Element.matches</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/Element/closest">Element.closest</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute">Element.toggleAttribute</a>
+            </li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key">KeyboardEvent.key</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver">IntersectionObserver</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver">ResizeObserver</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements">Custom
+              Elements</a></li>
+          </ul>
+        </div>
+      </div>
+      <p>All of them are fully supported by modern browsers, such as Chrome, Firefox, Safari or Edge (&gt;43).</p>
+      <p>In order to make ESL work in older browsers you can use a &quot;light&quot; polyfills list of IntersectionObserver,
+        ResizeObserver and Custom Elements
+        (Older versions of Edge and Safari)</p>
+      <p>Or make the library work in IE11 or Edge (&lt;14) by using the &quot;full&quot; polyfills list provided.</p>
+      <p>See more details on what polyfill approach might look like in the test-server examples.</p>
+      <p>Also, ESL has built-in polyfills for some of DOM and ES6 features. They are available under <a
+        href="https://github.com/exadel-inc/esl/tree/main/src/polyfills">./polyfills</a> directory.</p>
+    </div>
+    <hr/>
+
+    <h2 id="roadmap">Roadmap</h2>
+    <div class="container">
+      <ul class="m-8 mr-2 ml-2">
+        <li><a href="https://github.com/exadel-inc/ui-playground">UI Playground</a> (<em>will be available soon</em>) demo
```

Comment:
It seems to me that such content (playground URL) and the like needs to be inserted into templates as data so that you can easily change it later.
What do you think about it?

Reply:
I agree with you. This file will be rewritten soon, now we need it so that the start page is not empty